client/src/components/Add-Edit-Form/index.js

```diff
@@ -1,14 +1,108 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import { Form, Input, Button, message } from 'antd';
+import { Form, Input, Button, message, Spin } from 'antd';
 import './style.css';
 import AdminContainer from '../AdminContainer';
 
 const axios = require('axios');
 
-const AddEditForm = ({ formType, addLink, editLink, cohortId, inputData }) => {
-  const onFinish = async (values) => {
+const redirectFunc = (push, formType, cohortId, projectType) => {
```

Comment:
@MohammedAlghazali This redirect function could be moved inside the AddEditForm component class and then it could get push, formType and cohortId from props instead of requiring them as arguments.  That might be a little bit simpler.

Reply:
in this case, I can access to push, formType and cohortId from props but projectType this value come from sendValues (that come from <Form> when I make submit) so I can't access it if I make this function as a method,
what do you think

Reply:
ProjectType could still be an argument to the function - then it would just be the only argument.  For example, you'd call it like this:
`this.redirect(sendValues.projectType)`

Reply:
yes I can pass it as an argument,
I think I need some coffee 😁

Reply:
haha yes, me too